Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM alpine
+
+RUN apk --no-cache add gnupg \
+    build-base \
+    gettext-dev \
+    libgcrypt-dev \
+    zlib-dev \
+    gnutls-dev \
+    curl-dev \
+    cmake \
+    aspell-dev \
+    ncurses-dev
+
+RUN wget https://weechat.org/files/src/weechat-2.8.tar.xz \
+    && wget https://weechat.org/files/src/weechat-2.8.tar.xz.asc \
+    && gpg --keyserver hkps://keys.openpgp.org --recv-keys A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 \
+    # Check that the fingerprint is correct by checking the topic on #weechat channel (chat.freenode.net) 
+    # or by asking a developer.
+    && gpg --verify weechat-2.8.tar.xz.asc weechat-2.8.tar.xz \
+    && tar xvf weechat-2.8.tar.xz \
+    && mkdir -p weechat-2.8/build
```

Comment:
Considering this Dockerfile is in the development repository, shouldn't it use the source code from the cloned git repository, which this Dockerfile resides in instead of taking the last stable release code from weechat.org?

Reply:
If the Dockerfile uses sources from weechat.org, 2 remarks (to be discussed):

1. The file should be `weechat-stable.tar.xz` instead of `weechat-2.8.tar.xz`, there's always a link "stable" to the latest stable release.
2. Like many other projects, this Dockerfile could be moved to a separate repository inside the weechat organisation. Especially if we plan to add multiiple Docker files, for example development version or with different options: for example a container could be setup together with Glowing Bear to provide a graphical interface.

Reply:
Yes I think it's preferable to use `COPY` to commit the current files from git inside the container if the `Dockerfile` is to committed here.  I was using docker to test and build dependencies such as `gnutls` and so it could be nice to have a dockerfile for that included.  There are advantages to having a dockerfile in a separate repo using `weechat-stable.tar.xz` like being able to add to dockerhub and testing with other integrations (like glowing bear).

Reply:
I've updated this to `COPY` from the current git directory